clusters/build-clusters/01_cluster/hack/generate-bw-items.sh

```diff
@@ -55,7 +55,7 @@ generate_reg_auth_value() {
   config=$2
   local registry
   registry=$3
-  oc --kubeconfig "${config}" -n ci get secret $( oc --kubeconfig "${KUBECONFIG_BUILD01}" get secret -n ci -o "jsonpath={.items[?(@.metadata.annotations.kubernetes\.io/service-account\.name==\"build01\")].metadata.name}" ) -o "jsonpath={.items[?(@.type==\"kubernetes.io/dockercfg\")].data.\.dockercfg}" | base64 --decode | jq --arg reg "${registry}" -r '.[$reg].auth' > "${WORKDIR}/build01_${cluster}_reg_auth_value.txt"
+  oc --kubeconfig "${config}" -n ci get secret $( oc --kubeconfig "${KUBECONFIG_BUILD01}" get secret -n ci -o "jsonpath={.items[?(@.metadata.annotations.kubernetes\.io/service-account\.name==\"build01\")].metadata.name}" ) -o "jsonpath={.items[?(@.type==\"kubernetes.io/dockercfg\")].data.\.dockercfg}" | base64 --decode | jq --arg reg "${registry}" -r '.[$reg].auth' | tr -d '\n' > "${WORKDIR}/build01_${cluster}_reg_auth_value.txt"
```

Comment:
This made a line break at the end of the txt file (now we remove it).
The content of this file will be in a secret which is used by `applyconfig`, which is an env. var. for rendering values of a template.
Since it is a secret, we censor it in the log.
Which would cause the deadlock of logrus as we discussed https://github.com/kubernetes/test-infra/pull/15935